.github/workflows/ci-backend.yml

```diff
@@ -0,0 +1,35 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI_BACKEND
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the development branch
+  push:
+    branches: [ development ]
+  pull_request:
+    branches: [ development ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
```

Comment:
should we always do latest, or should we specify a specific version that matches with our backend hosting?

Reply:
Good question, @Tahmeed27 what do you think?

Reply:
https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners. This is a list of the host runners github provides.

Reply:
I'm down to make it latest/ubuntu-20.04 for now, and have a ticket to update it when we have deployments running

Reply:
Yeah I say keep it latest for now